### PR TITLE
Add 0xj4an as delegate

### PR DESCRIPTION
**Pull request by work-0xj4an**

### CODE REVIEW

**Comment by aaronmgdr** [comment]
@work-0xj4an happy to approve once we have the prettier error fixed

**Review by aaronmgdr — CHANGES_REQUESTED** [review]
you will need to change the file name of the image. it looks like it currently has spaces in the name.

**Comment by jmrossy** [comment]
+1 to @aaronmgdr's comments

File: src/config/delegates.json [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+    "name": "0xj4an",
+    "address": "0xFEF5A1A2b3754A2F53161EaaAcb3EB889F004d4a",
+    "logoUri": "/logos/delegatees/0xj4an.jpg",
+    "date": "2024-08-03",
+    "links": {
+      "website": "https://linktr.ee/0xj4an_work",
+      "twitter": "https://twitter.com/0xj4an"
+    },
+    "interests": ["Community", "Transparency", "Accountability", "Good Vibes"],
+    "description": "Hey im I am a Celo Guardian (Old named CGP Editors), adding myself as delegate"
```
Comment (jmrossy) [inline src/config/delegates.json:12]:
```suggestion
    "description": "Hey, I am a member of the Celo Guardians (formerly known as 'CGP Editors'). I help maintain the history of governance proposals and am an active contributor to the Celo community."
```

**Review by jmrossy — APPROVED** [review]

**Pull request merged**